apps/condo/domains/acquiring/access/GeneratePaymentLinkService.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Generated by `createservice acquiring.GeneratePaymentLinkService`
+ */
+async function canGeneratePaymentLink ({ authentication: { item: user } }) {
+    return true
```

Comment:
Check for user is actually exists. I think anonymous user shouldn't has access to this method

Reply:
ok